### ISSUE TITLE
%S emits null rather than an empty string

### ISSUE THREAD

**Issue by jollytoad** [stephenh/ts-poet#1]
`CodeBlock.of('%S', '').toString()` emits `null`

I'd expect it to emit a quoted empty string: `""`

I think this is down to https://github.com/stephenh/ts-poet/blob/master/src/CodeWriter.ts#L355 ...

```
private emitString(s?: string): void {
  // Emit null as a literal null: no quotes.
  this.emit(s ? stringLiteralWithQuotes(s) : 'null');
}
```

this should probably check `s !== null`

**Comment by stephenh** [comment]
Thanks for the bug report! This should be fixed in 0.5.5.

Let me know if you run into other issues as well, either out right bugs or other UX-type issues.

**Comment by jollytoad** [comment]
Awesome, thank you very much.
